src/adapters/sqlite/index.js

```diff
@@ -286,17 +264,18 @@ export default class SQLiteAdapter implements DatabaseAdapter {
     this._batch(
       [
         [
-          'execute',
+          0,
```

Comment:
An descriptive enum (eg. `CacheBehavior.Ignore`) might help newcomers to grasp what's going on here without the need to delve into the implementation

Reply:
I did something similar to your suggestion in #1040… I don't like the exported enum pattern in JavaScript because it unnecessarily adds to the code size :/ But I defined locally named constants to help the reader understand what's going on.